ebpf.go

```diff
@@ -73,6 +73,11 @@ type Map interface {
 	Update(interface{}, interface{}) error
 	Upsert(interface{}, interface{}) error
 	Delete(interface{}) error
+	// Implementation of bpf_map_get_next_key
+	GetNextKey(interface{}) ([]byte, error)
+	GetNextKeyString(interface{}) (string, error)
```

Comment:
while refactoring the tests, i came across an issue where the string wasn't properly formatted just by casting it.
then i mimicked the interface to handle both string and int values for the key.

in the code for the Lookup, i see comments like:

```
// LookupString perform lookup and returns GO string from NULL terminated C string
// WARNING: Does NOT work for Per-CPU maps (not an real use case?).
--
// LookupUint64 performs lookup and returns uint64
// WARNING: For Per-CPU array/hash returns sum of values from all CPUs
```

i am not familiar enough with ebpf and other map types to understand if the behavior of GetNextKey should be different for other map types. what are your thoughts on this?